update-integration-configs/task

```diff
@@ -61,6 +61,14 @@ function main() {
       new_cats_integration_config=$(echo "${new_cats_integration_config}" | jq ".api=\"api.${SYSTEM_DOMAIN}\" | .apps_domain=\"${SYSTEM_DOMAIN}\"")
     fi
 
+    if [ -n "${PRIVATE_DOCKER_REGISTRY_IMAGE}" ]; then
```

Comment:
Could you please update the uses of single square brackets to double square brackets in this file? (I know you didn't introduce this)